sktime/transformations/series/compose.py

```diff
@@ -96,3 +97,210 @@ def inverse_transform(self, Z, X=None):
         if not self.passthrough:
             z = self.transformer_.inverse_transform(z, X=None)
         return z
+
+
+class ColumnwiseTransformer(_SeriesToSeriesTransformer):
+    """
+    Applies a transformer for univariate series
+    columnwise to multivariate series.
+
+    Parameters
+    ----------
+    transformer : Estimator
+        scikit-learn-like or sktime-like transformer to fit and apply to series
+    columns : list of str
+            Names of columns that are supposed to be transformed
+
+     Attributes
+    ----------
+    transformers_ : dict of {str : transformer}
+        Maps columns to transformers
+
+    Example
+    -------
+    >>> from sktime.datasets import base
+    >>> from sktime.transformations.series.detrend import Detrender
+    >>> from sktime.transformations.series.compose import ColumnwiseTransformer
+
+    >>> y, X = base.load_longley()
+    >>> transformer = ColumnwiseTransformer(Detrender())
+    >>> yt = transformer.fit_transform(X)
+    """
+
+    _required_parameters = ["transformer"]
+
+    def __init__(self, transformer, columns=None):
+        self.transformer = transformer
+        self.columns = columns
+
+        # check that columns are None or list of strings
+        if columns is not None:
+            if not isinstance(columns, list) and all(
+                isinstance(s, str) for s in columns
+            ):
+                raise ValueError("Columns need to be a list of strings or None.")
+
+        self.columns_ = columns
+        self.transformers_ = None
```

Comment:
I am wondering if a trailing underscore is required here or if we should rather make it a private variable like `_columns` 
 and `_transformers`? I cant think of a use case where you would want to get a particular transformer after fitting. Also it seems e.g. `EnsembleForecaster` does not have sth like `self.forecaster_` where the parts of the ensemble are exposed.

Reply:
I'm not sure about this one. I looked at other transformers and for example in `_detrend` there is also a `self.forecaster_` and in adapt there is a `self.transformer_`. Maybe @mloning can clarify?

Reply:
ok, then in my opinion lets keep it here also to be consistent